sample_project/Assets/SampleViewer/Samples/FeatureLayer/Scripts/FeatureLayerUIManager.cs

```diff
@@ -8,22 +14,23 @@
 public class FeatureLayerUIManager : MonoBehaviour
 {
     [SerializeField] private Animator dropDownAnim;
-    [SerializeField] private Toggle dropDownButton;
+    public Toggle dropDownButton;
```

Comment:
For organization, I would move these variables that you made `public` above the private fields with a space between them.

```suggestion
    public Toggle DropDownButton;
```